### DISCUSSION TITLE
Does LoRA PEFT Integration work for inpainting models?

### DISCUSSION THREAD

**Discussion by Scorpinaus** [huggingface/diffusers#7587] [Q&A]
Hi diffusers team!

Have checked the docs regarding PEFT LORA Integration, where its used for inference. Does the peft integration for LoRAs work for inpainting pipeline?

**Comment by tolgacangoz** [comment]
Hi @Scorpinaus!
For the pipelines inheriting from `LoraLoaderMixin` or `StableDiffusionXLLoraLoaderMixin`, which have [`USE_PEFT_BACKEND` checks](https://github.com/huggingface/diffusers/blob/6133d98ff70eafad7b9f65da50a450a965d1957f/src/diffusers/loaders/lora.py#L108-L109) in them, yes. You can see, for instance, [`StableDiffusionInpaintPipeline` inheriting from `LoraLoaderMixin`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py#L226). The inpainting pipeline takes LoRA-related functions.
Does this answer your question, or is there something you would like me to elaborate on?

**Comment by sayakpaul** [comment]
@standardAI thanks a ton for being so helpful with our community members. Appreciate you spending a considerable amount of time answering these questions. ❤️
